hype-submitter/src/main/java/com/spotify/hype/model/RunEnvironment.java

```diff
@@ -32,29 +32,13 @@
 @AutoMatter
 public interface RunEnvironment {
 
-  EnvironmentBase base();
+  Optional<Path> yamlPath();
   List<Secret> secretMounts();
   List<VolumeMount> volumeMounts();
   Map<String, String> resourceRequests();
 
-  interface EnvironmentBase {
-  }
-
-  @AutoMatter
-  interface SimpleBase extends EnvironmentBase {
-    String image();
-  }
-
-  @AutoMatter
-  interface YamlBase extends EnvironmentBase {
-    Path yamlPath();
-    Optional<String> overrideImage();
-  }
-
-  static RunEnvironment environment(String image) {
-    return new RunEnvironmentBuilder()
-        .base(new SimpleBaseBuilder().image(image).build())
-        .build();
+  static RunEnvironment get() {
```

Comment:
I think this method should be renamed to `environment()`. When you import it statically `get()` is pretty confusing to read